_includes/icons/icons.html

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+  {% include icons/link.html %}
+  {% include icons/menu.html %}
+  {% include icons/expand.html %}
+  {% include icons/external_link.html %}
+  {% if site.search_enabled != false %}
+    {% include icons/document.html %}
+    {% include icons/search.html %}
+  {% endif %}
+  {%- if site.enable_copy_code_button != false %}
```

Comment:
Is the hyphen in `{%-` necessary to suppress some whitespace for correctness? The theme uses [penibelst/jekyll-compress-html](https://github.com/penibelst/jekyll-compress-html) to compress whitespace for all output pages.

Reply:
Ah no, this was an oversight on my part - thanks!